### ISSUE TITLE
focus rings overlap alert messages. 

### ISSUE THREAD

**Issue by akomiqaia** [Financial-Times/origami#942]
There was [some discussion](https://financialtimes.slack.com/archives/C02FU5ARJ/p1671631596586179?thread_ts=1671550538.570259&cid=C02FU5ARJ) about making an extra bit of space between the alert message and input element so focus ring would not obscure it. 
<img width="634" alt="Screenshot 2022-12-21 at 14 18 53" src="https://user-images.githubusercontent.com/31027725/209128279-7a30a1b4-670e-4af3-aee7-9c4f108a36ed.png">

**Comment by notlee** [comment]
Note: In our demos there's no overlap but the focus ring does run right against the error message.

**Comment by notlee** [comment]
Also note the clipping of the focus ring overflow is a product specific issue and not related to o-overlay.
